integration-tests/src/server.js

```diff
@@ -7,9 +7,9 @@ const { testFindWorkflow } = require('../tests/find-workflow')
 const { testImportCohortData } = require('../tests/import-cohort-data')
 const { testImportDockstoreWorkflow } = require('../tests/import-dockstore-workflow')
 const { testRegisterUser } = require('../tests/register-user')
-const { testRunNotebook } = require('../tests/run-notebook')
+//TODO IA-3436 const { testRunNotebook } = require('../tests/run-notebook')
```

Comment:
FYI, I haven't been adding any of my tests to this file, as I don't believe it's used for anything. It would be good to delete all this Bueller stuff.